app/api/controllers/RequestHandler.scala

```diff
@@ -123,21 +126,34 @@ object RequestHandler {
 
       implicit class Response(result: Result) {
 
+        private val HttpDateFormatter = DateTimeFormatter
+          .ofPattern("EEE, dd MMM yyyy HH:mm:ss z", Locale.ENGLISH)
+          .withZone(ZoneId.of("GMT"))
+
         def withApiHeaders(correlationId: String, responseHeaders: (String, String)*)(implicit appConfig: AppConfig, apiVersion: Version): Result = {
           val maybeDeprecatedHeader =
             if (appConfig.isApiDeprecated(apiVersion))
               List(
-                "Deprecation" -> s"This endpoint is deprecated. See the API documentation: ${appConfig.apiDocumentationUrl}"
+                "Deprecation" -> s"${HttpDateFormatter.format(appConfig.deprecatedOn(apiVersion).get)}",
+                "Link"        -> s"${appConfig.apiDocumentationUrl}"
               )
             else Nil
 
+          val maybeSunsetHeader =
+            if (appConfig.sunsetDate(apiVersion).nonEmpty) {
+              List("Sunset" -> s"${HttpDateFormatter.format(appConfig.sunsetDate(apiVersion).get)}")
+            } else if (appConfig.sunsetDate(apiVersion).isEmpty && appConfig.sunsetEnabled(apiVersion)) {
+              List("Sunset" -> s"${HttpDateFormatter.format(appConfig.deprecatedOn(apiVersion).get.plusMonths(6))}")
+            } else Nil
+
           val headers =
             responseHeaders ++
               List(
                 "X-CorrelationId"        -> correlationId,
                 "X-Content-Type-Options" -> "nosniff"
               ) ++
-              maybeDeprecatedHeader
+              maybeDeprecatedHeader ++
```

Comment:
as above with separating out, this can be a function call to return both in one Seq, 'maybeDeprecationHeaders' or sm